lib/watcher.js

```diff
@@ -58,7 +58,9 @@ module.exports = class Watcher extends EventEmitter {
       logger.debug('change', 'ignored: rebuild scheduled already');
       return;
     }
-    logger.debug('change');
+    logger.debug('change', event, filePath, root);
+    this.emit('change', event, filePath, root);
```

Comment:
If I remember correctly the "change" event that ember-cli is expecting is a "results" object that contains: `directory`, `filepath`, `graph`, `outputChanges`, `totaltime`. Is the plan to modify that side to include just the above properties or should we emit the ones that ember-cli is expecting?

Reply:
Correct, and tbh I think that API isn’t good. A “change” event in ember-cli-sane-watcher indicates a build completed, as it does its own change tracking for individual files but doesn’t expose those events (add,change,delete). 

I am proposing we expose the change events and separate them from the build events. You can see the changes here https://github.com/ember-cli/ember-cli/issues/8614